ui-super-heroes/pom.xml

```diff
@@ -6,4 +6,137 @@
   <artifactId>ui-super-heroes</artifactId>
   <version>1.0</version>
   <name>Quarkus Sample :: Super-Heroes :: User Interface</name>
+  <properties>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.release>17</maven.compiler.release>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+    <quarkus.platform.version>3.0.3.Final</quarkus.platform.version>
+    <skipITs>true</skipITs>
```

Comment:
Should remove this to be consistent with the rest of the apps (we ALWAYS run integration tests).